content/post/My-journey-with-ALiAS.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "My journey with ALiAS : From a commit to the community"
+slug = "animesh-alias-journey"
+image = "/images/community.jpg"
+date = "2019-05-25T00:00:00"
+tags=['ALiAS, FOSS, Alumni, Experiences']
+categories = ['Alumni Journals']
```

Comment:
```suggestion
categories = ['Alumni-Journals']
```
The blog is facing some issues with 2 word strings, hence a workaround. 
CC: @Sc0rpi0n101

Reply:
Done.